index.js

```diff
@@ -622,6 +638,72 @@ var QUERIES = {
       }, [])
     }
   },
+  baseline: {
+    matches: ['year', 'availability'],
+    regexp: /^baseline\s+(?:(\d+)|(newly|widely)\s+available)$/i,
+    select: function (context, node) {
```

Comment:
This function is a bit gnarly but it also reflects the underlying complexity that is coming from the baseline definitions.

I am happy to apply any suggestions :)

Reply:
The current code is too complex, yes.

What do you think if we do something like:

```js
let since = `2020-01-15` // generate the correct date
return resolve(`since ${since}`).map(browser => BASELINE_BROWSERS[browserName(browser)])
```